src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3485,6 +3485,28 @@ internal void SymbolDeclaredEvent(Symbol symbol)
             EventQueue?.TryEnqueue(new SymbolDeclaredCompilationEvent(this, symbol.GetPublicSymbol()));
         }
 
+        internal override ImmutableDictionary<string, string> SerializeForPdb()
+        {
+            var builder = ImmutableDictionary.CreateBuilder<string, string>();
```

Comment:
Please use an explicit comparer here. Also why are you chosing ordinal here when options are generally case insensitive

Reply:
Going to change to take blobbuilder and write directly instead of using ImmutableDictionary